atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/expectations/ExpectationsAwareTransaction.java

```diff
@@ -16,16 +16,19 @@
 
 package com.palantir.atlasdb.transaction.api.expectations;
 
-import com.google.common.collect.ImmutableMap;
-import com.palantir.atlasdb.keyvalue.api.TableReference;
-import org.immutables.value.Value;
+import com.palantir.atlasdb.transaction.api.Transaction;
 
```

Comment:
Since this is an entry point into the world of TEX, lets leave a interface level javadoc explaining what expectations are and how people should use it